packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -177,8 +177,18 @@ export default class ConfigParser {
     private merge(object: MergeConfig = {}, addPathToSpecs = true) {
         const spec = Array.isArray(object.spec) ? object.spec : []
         const exclude = Array.isArray(object.exclude) ? object.exclude : []
-        this._config = deepmerge(this._config, object) as TestrunnerOptionsWithParameters
-
+        const customDeepMerge = deepmergeCustom({
+            mergeArrays: (values, utils, meta) => {
+                if (meta && ['services', 'reporters'].includes(meta.key as string)) {
```

Comment:
Experiencing a similar issue when passing `capabilities` into `Launcher` (capabilities are duped due to `deepmerge`). Here is a reproducible example: https://github.com/EdmundasD/testing-wdio-v8
It appears that adding `capabilities` to the array here resolves that issue, after doing some preliminary testing.

Reply:
@EdmundasD can you verify if this is fixed when applying the changes in this PR?

Reply:
Yeap, that's exactly what I did. It resolved the issue I was having and passing the capabilities directly in the config also works as expected.

Reply:
Thanks for verifying.

Reply:
@christian-bromann, to be clear, `capabilities` need to be added to the array here in order to resolve the issue I described:
```
...
if (meta && ['services', 'reporters', 'capabilities'].includes(meta.key as string)) {
...
```
cc @jemishgopani

Reply:
@christian-bromann Do we need this change in this PR?

Reply:
I think we can include it. I doubt it will blow up the scope but I will let you decide.